### DISCUSSION TITLE
HEFLO para la parte del BPM en los requerimientos de software

### DISCUSSION THREAD

**Discussion by roge64** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#108] [Show and tell]
Buenos días, 

Les comparto este recurso para que puedan generar su diagrama de BPM (Gestión de Procesos de Negocio), no es el único pero se encuentra a la mano.

https://www.heflo.com/

y un video para aprender a usarlo de forma rápida con ejemplo sencillo.

https://www.youtube.com/watch?v=ft_4Fl-WMuY&t=112s

Espero les sea de utilidad.

**Comment by RodolfoMorquecho** [comment]
Te refieres a la parte de requerimientos del sistema, cierto?

**Reply by roge64** [reply]
En la parte 3. Análisis de Requisitos y Requerimientos
